bin/yarn

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+APP_ROOT = File.expand_path('..', __dir__)
+Dir.chdir(APP_ROOT) do
+  begin
+    exec "yarnpkg", *ARGV
+  rescue Errno::ENOENT
+    $stderr.puts "Yarn executable was not detected in the system."
+    $stderr.puts "Download Yarn at https://yarnpkg.com/en/docs/install"
+    exit 1
+  end
+end
```

Comment:
Only small comment from me is that in my suggestion I avoided setting the `APP_ROOT` constant because it felt a bit confusing to me (we're not an app), and I don't really expect developers to run this binstub from the root of the repo.

I'm happy with the current version though if you disagree.